tests/cephfs/cephfs_utils.py

```diff
@@ -65,6 +65,89 @@ def _setup_crefi(node):
         # Setup Crefi pre-requisites : pyxattr
         node.exec_command(sudo=True, cmd="pip3 install pyxattr", long_running=True)
 
+    def prepare_clients(self, clients, build):
+        for client in clients:
+            node = client.node
+            if node.pkg_type == "rpm":
+                out, rc = node.exec_command(
+                    sudo=True, cmd="rpm -qa | grep -w 'attr\\|xattr'"
+                )
+                output = out.read().decode()
+                output.split()
+                if "attr" not in output:
+                    node.exec_command(sudo=True, cmd="yum install -y attr")
+                node.exec_command(
+                    sudo=True, cmd="yum install -y gcc python3-devel", check_ec=False
+                )
+
+                if "xattr" not in output:
+                    pkgs = [
```

Comment:
Would it make sense to move the package installation to a different method? This method looks to be big... I am thinking if we could simplify it a bit.